enterprise_access/apps/subsidy_request/admin.py

```diff
@@ -0,0 +1,88 @@
+""" Admin configuration for subsidy_request models. """
+
+from django.contrib import admin
+from django.utils.translation import gettext_lazy as _
+
+from enterprise_access.apps.subsidy_request import models
+
+
+class SubsidyRequestStatusFilter(admin.SimpleListFilter):
+    """ Filter requests by approval status. """
+
+    title = _('approval status')
+
+    parameter_name = 'approval_status'
+
+    def lookups(self, request, model_admin):
+        return (
+            ('approved', _('Approved')),
+            ('pending', _('Pending')),
```

Comment:
Chris, I want to [draw your attention here](https://openedx.atlassian.net/wiki/spaces/SOL/pages/3266543926?focusedCommentId=3313303563#comment-3313303563), we will need an additional state for approved pending assignment (The state when the admin as approved but the code/sub has not yet been assigned).

Reply:
My other question on this would be by injecting this new state, should we add a state field to the request object to track said states?

Reply:
Currently we don't have a state field and the state is computed from the approved_at/denied_at field. We could add a state but I think we could get away without it.

Reply:
hmm i might want to change this a bit

Reply:
going to consolidate approved_at/denied_at to reviewed_at and add a state field